nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -269,6 +269,19 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - `paperless`' `services.paperless.extraConfig` setting has been removed and converted to the free-form type and option named `services.paperless.settings`.
 
+- `apptainer` and `singularity` now prioritize `PATH` from the system over those constructed from dependent packages, when it comes to the substitution of `defaultPath` values. It is now constructed by the following sources, ordered by their precedence:
```

Comment:
To be honest, at this point I don't have a good idea of what `defaulPath` refers to, I'm not sure an average reader does either